app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java

```diff
@@ -416,6 +416,14 @@ private void exitFullScreenMode() {
             return;
         }
 
+        // We need to add a delay for the exitFullScreen() call to solve some viewport scaling issues,
+        // See https://github.com/MozillaReality/FirefoxReality/issues/833 for more info.
+        getHandler().postDelayed(() -> {
+            if (SessionStore.get().isInFullScreen()) {
+                SessionStore.get().exitFullScreen();
+            }
+        }, 50);
```

Comment:
this solves the issue for now. shall we keep #833 open for a more proper investigation + solution after 1.1?